pages/common/pio-lib.md

```diff
@@ -0,0 +1,36 @@
+# pio lib
+
+> Manage PlatformIO libraries.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/lib/>.
+
+- List installed libraries:
+
+`pio lib list`
+
+- List built-in libraries based on installed development platforms and their frameworks:
+
+`pio lib buintin`
+
+- Search for existing libraries:
+
+`pio lib search {{library}}`
+
+- Show details about a library:
+
+`pio lib show {{library}}`
+
+- Install a library:
+
+`pio lib install {{library}}`
+
+- Update installed libraries:
+
+`pio lib update`
+
+- Uninstall a library:
+
+`pio lib uninstall {{library}}`
+
+- Show PlatformIO library registry statistics:
```

Comment:
Is this local stats or remote stats?

Reply:
As far as I know, there is only a remote registry.

Reply:
According to the [documentation](https://docs.platformio.org/en/v5.1.1/core/userguide/lib/cmd_stats.html), it shows the versions of the PlatformIO registry libs, i.e. remote.